APCAppCore/APCAppCore/Library/Objects/APCTasksReminderManager.h

```diff
@@ -33,11 +33,42 @@
  
 #import <Foundation/Foundation.h>
 #import "APCTaskReminder.h"
+
+// These must be reflected by NSCalendar NSCalendarUnitWeekday, except for the EveryDay value
+static NSUInteger const kAPCTaskReminderDayOfWeekEveryDay    = 0;
+static NSUInteger const kAPCTaskReminderDayOfWeekSunday      = 1;
+static NSUInteger const kAPCTaskReminderDayOfWeekMonday      = 2;
+static NSUInteger const kAPCTaskReminderDayOfWeekTuesday     = 3;
+static NSUInteger const kAPCTaskReminderDayOfWeekWednesday   = 4;
+static NSUInteger const kAPCTaskReminderDayOfWeekThursday    = 5;
+static NSUInteger const kAPCTaskReminderDayOfWeekFriday      = 6;
+static NSUInteger const kAPCTaskReminderDayOfWeekSaturday    = 7;
+
 @interface APCTasksReminderManager : NSObject
 @property (nonatomic) BOOL reminderOn;
 @property (nonatomic, strong) NSString * reminderTime; //Should be an element of reminderTimesArray
 @property (strong, nonatomic, getter=reminders) NSMutableArray *reminders;
 
+/**
+ * The days of the week to repeat the reminder, defaults to include every day
+ * Object in the array must be of type APCTaskReminderDayOfWeek
+ */
+@property (nonatomic, strong) NSArray* daysOfTheWeekToRepeat;
+
+/*
+ * If true, all local notifications in this app (even ones that are not reminders) will be removed
+ * If false, the app will try and find only reminder notifications to delete; however this has inconsistant behavior
+ */
+@property (nonatomic) BOOL updatingRemindersRemovesAllLocalNotifications;
+
+/*
+ * Update reminder messaging
+ * This defualts to...
+ * "Please complete your TeamStudy activities today. Thank you for participating in the TeamStudy study!"
```

Comment:
it uses appname rather than TeamStudy right? should probably refer to appname rather than TeamStudy in appcore code.